frontend/components/ClubDisplay.tsx

```diff
@@ -58,6 +60,71 @@ const ClubDisplay = ({
     )
   }
 
+  if (ranked) {
+    // Separate clubs by tier property
+    const tieredClubs = displayClubs.reduce((acc, club) => {
+      const tier = club.tier || ' '
+      if (!acc[tier]) {
+        acc[tier] = []
+      }
+      acc[tier].push(club)
+      return acc
+    }, {})
+
+    const tierColors = {
+      S: 'salmon', // Paler light red
+      A: 'peachpuff', // Paler light orange
+      B: 'lightgoldenrodyellow', // Paler light yellow
+      C: 'palegreen', // Paler light green
+      D: 'powderblue', // Paler light turquoise
+      E: 'plum', // Paler light purple
+      F: 'mistyrose', // Paler light pink
+      ' ': '#CCCCCC',
+    }
+
+    return (
+      <ClubTableRowWrapper>
+        {Object.entries(tieredClubs).map(([tier, clubs]: [string, Club[]]) => (
+          <div
+            key={tier}
+            style={{
+              borderRadius: '10px',
+              backgroundColor: tierColors[tier],
+              position: 'relative',
+              marginTop: '20px',
+            }}
+          >
+            {tier !== ' ' && (
+              <div
+                style={{
+                  position: 'absolute',
+                  top: '4px',
+                  right: '4px',
+                  width: '36px',
+                  height: '36px',
+                  borderRadius: '6px',
+                  backgroundColor: '#FFFFFF33',
+                  textAlign: 'center',
+                  verticalAlign: 'middle',
+                  lineHeight: '36px',
+                  fontWeight: 'bold',
+                  fontSize: '1.2rem',
+                }}
+              >
+                {`${tier}`}
```

Comment:
I will not allow this

Reply:
![image](https://github.com/pennlabs/penn-clubs/assets/10285518/cb6057ed-6ba0-4829-b395-40240c6d6b25)